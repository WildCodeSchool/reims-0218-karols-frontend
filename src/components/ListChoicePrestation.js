import React from "react"
import ChoicePrestation from "./ChoicePrestation"
import { Col, Row } from "reactstrap"

const ListChoicePrestation = () => (
  <Row className="text-center">
    <Col xs="12" md="6" lg="3">
      <ChoicePrestation />
    </Col>
    <Col xs="12" md="6" lg="3">
      <ChoicePrestation />
    </Col>
    <Col xs="12" md="6" lg="3">
      <ChoicePrestation />
    </Col>
    <Col xs="12" md="6" lg="3">
      <ChoicePrestation />
    </Col>
  </Row>
)

export default ListChoicePrestation
