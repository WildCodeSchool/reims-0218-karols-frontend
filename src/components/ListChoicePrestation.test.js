import React from "react"
import ReactDOM from "react-dom"
import ListChoicePrestation from "./ListChoicePrestation"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ListChoicePrestation />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<ListChoicePrestation />).toJSON()
  expect(tree).toMatchSnapshot()
})
