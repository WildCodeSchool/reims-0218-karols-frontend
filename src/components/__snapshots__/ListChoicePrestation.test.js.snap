// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <hr />
    <h2
      className="display-4"
    >
      Etape 4: SÃ©lectionnez vos prestations
    </h2>
  </div>
</div>
`;
